### DISCUSSION TITLE
archive.org videos can't be played on qutebrowser?

### DISCUSSION THREAD

**Discussion by amano-kenji** [qutebrowser/qutebrowser#7324] [Q&A] [answered]
For example, https://archive.org/details/solutionswatch-childrenhistory

The error message is

> This video cannot be played because of a technical error.
(Error Code: 100000)

**Comment by The-Compiler** [comment, marked as answer]
Can you reproduce with `--temp-basedir`? Can you please show your full `:version` information?

**Reply by amano-kenji** [reply]
It turns out archive.org requires canvas for video playback. I disabled canvas reading in qutebrowser to minimize browser fingerprinting.

I wish qutebrowser randomized canvas fingerprint.
